synapse/cores/common.py

```diff
@@ -803,6 +818,23 @@ def getSizeBy(self, name, prop, valu, limit=None):
         meth = self._reqSizeByMeth(name)
         return meth(prop,valu,limit=limit)
 
+    def initTufosBy(self, name, meth):
+        '''
+        Initialize a "tufos by" handler for the Cortex.  This is useful
+        when the index or storage backing can optimize tufo creation from
+        raw rows.
+
+        Example:
+            def getbywoot(prop,valu,limit=None):
+                return stuff() # list of tufos
+
+            core.initTufos('woot',getbywoot)
```

Comment:
as a casual user of synapse/cortex, this example is not very helpful to me. the sample is too generic (`woot` is a weird index ;-) ) and the impl of  `stuff()` is the thing that is really interesting, but not defined here.

could we provide the naive `in` implementation here as a better example? its only a few lines longer, and could be used in the real world.